research/object_detection/meta_architectures/context_rcnn_meta_arch.py

```diff
@@ -264,11 +264,17 @@ def __init__(self,
             return_raw_detections_during_predict),
         output_final_box_features=output_final_box_features)
 
-    self._context_feature_extract_fn = functools.partial(
-        context_rcnn_lib.compute_box_context_attention,
-        bottleneck_dimension=attention_bottleneck_dimension,
-        attention_temperature=attention_temperature,
-        is_training=is_training)
+    if tf_version.is_tf1():
+      self._context_feature_extract_fn = functools.partial(
+          context_rcnn_lib.compute_box_context_attention,
+          bottleneck_dimension=attention_bottleneck_dimension,
+          attention_temperature=attention_temperature,
+          is_training=is_training)
```

Comment:
to unify the interface between the two options, leave out is_training from partial call.